launch/passive_stack.launch

```diff
@@ -16,7 +16,7 @@
     <include if="$(eval str(arg('platform')) != '1.5')"
              file="$(find iarc7_sensors)/launch/obstacle_detector.launch" />
 
-    <include if="$(eval str(arg('platform')) != '1.5')"
+    <include if="$(eval str(arg('platform')) != '1.5' and str(arg('platform')) != 'sim_1.5')"
```

Comment:
What's the deal with this?  Why are there two versions of this param?

Reply:
One of them is for 1.5's sim and one of them is for the drone called 1.5.

Reply:
I'm not a huge fan, eventually one of them will go away.